stable/xray/templates/xray-statefulset.yaml

```diff
@@ -120,6 +120,15 @@ spec:
       - name: {{ .Values.router.name }}
         image: '{{ .Values.router.image.repository }}:{{ default .Chart.AppVersion .Values.router.image.version }}'
         imagePullPolicy: {{ .Values.router.image.imagePullPolicy }}
+        command:
+          - '/bin/sh'
```

Comment:
Change `sh` to `bash` (make sure we have it in the image first).

Reply:
Hi Eldada I copied the same configuration that all the other xray prestart command use. Can you explain why you want just this one to be different than the other 4? This is not the artifactory chart.

Reply:
It's needed to properly pass SIGTERM to entrypoint. Indeed - this should be for all containers. I will send you more details in a DM. Leave this for now.